plasmapy/utils/decorators/helpers.py

```diff
@@ -142,3 +142,51 @@ def wrapper(*args, **kwargs):
         wrapper.__signature__ = inspect.signature(f)
 
     return wrapper
+
+
+def add_lite(flite, attrs=None, scope=None):
+    """
+    Decorator to bind lightweight "lite" versions of formulary functions to the full
+    formulary function.
+    """
+    if attrs is None:
+        attrs = []
+    elif scope is None:
```

Comment:
Is there a reason to not just set scope=globals() by default?

Reply:
I suspect this part could be optimized out given some tinkering. Maybe `sys.modules(__module__)` combined with `importlib` and `inspect.getmembers` in some funny way...

But at this point, if you just bluntly set `globals` as a default from `helpers.py`, it's going to "inherit the scope of helpers.py" rather than the local environment of the function that's being registered.

Reply:
Got it - that makes sense (as much as anything regarding scopes does)

Reply:
> But at this point, if you just bluntly set globals as a default from helpers.py, it's going to "inherit the scope of helpers.py" rather than the local environment of the function that's being registered.

Yes, this is what was happening when `globals()` was the default value.